src/main/java/org/spongepowered/api/entity/living/player/server/ServerPlayer.java

```diff
@@ -32,4 +33,6 @@
     @Override
     ServerWorld getWorld();
 
+    Server getServer();
```

Comment:
ServerWorld has a getServer I believe.

Reply:
Yes that is true. This was mainly a thought from vanilla having a getServer method on it's own server player entity